source/connect-zbt1/index.html

```diff
@@ -175,32 +181,35 @@ <h3>Form Factor</h3>
+
   {% details "Why include a USB extension cable?" %} USB 3.0 ports (the ones with blue on the inside) are known to cause significant noise and radio interference to any 2.4Ghz  wireless devices. This includes Zigbee and Thread. If you do not use the extension cable, it may not work at all, and if it does, it could be flaky at best with intermittent problems (issues with pairing, device dropouts, unreachable devices, timeout errors, etc).
   <br><br>
   <a href="https://www.youtube.com/watch?v=tHqZhNcFEvA">See demo on YouTube.</a> 
   {% enddetails %}
 
-  {% details "Which use case do you recommend for Home Assistant SkyConnect?" %}
-  Our current recommendation for Home Assistant SkyConnect is to use the Zigbee firmware to power your Zigbee network. This solution offers a great experience out-of-the-box.<br /><br />
+  {% details "Which use case do you recommend for Home Assistant Connect ZBT-1?" %}
+  Our current recommendation for Home Assistant Connect ZBT-1 is to use the Zigbee firmware to power your Zigbee network. This solution offers a great experience out-of-the-box.<br /><br />
   As part of our efforts to bring Matter to Home Assistant, we're now focused on ensuring that the Thread experience will catch up and become a first-class citizen, making it easy to connect to your Thread-based Matter devices in Home Assistant without a third-party Thread border router. The Thread firmware is already fully functional under the hood, but we still have some work to do to make the experience of using Thread-based devices in Home Assistant feel good. Once the experience has improved, we will recommend using this Thread firmware to power your Thread network as an alternative to using third-party Thread border routers from Apple or Google.
   {% enddetails %}
 
   {% details "What is the current state of multiprotocol support?" %} 
   <br />
   <b>Multiprotocol will remain an experimental feature and is not recommended for use. </b>
   <br /><br />
-  When we launched Home Assistant SkyConnect, we announced our intent to release firmware supporting multiprotocol, which allows the device's Silicon Labs chip to connect to both Zigbee and Thread networks with one radio. <br /><br />
-  This experimental firmware has been available since December 2022. Through extensive testing, we have found that although it works in some circumstances, it has technical limitations that lead to a worse user experience. We now do not recommend using this firmware, and it will be experimental for the foreseeable future. Instead, we will focus on making sure the dedicated Zigbee and Thread firmwares for Home Assistant SkyConnect deliver the best experience to users.<br /><br />
+  When we launched Home Assistant SkyConnect (Now known as Home Assistant Connect ZBT-1), we announced our intent to release firmware supporting multiprotocol, which allows the device's Silicon Labs chip to connect to both Zigbee and Thread networks with one radio. <br /><br />
```

Comment:
```suggestion
  When we launched Home Assistant SkyConnect (Now known as Home Assistant Connect&nbsp;ZBT-1), we announced our intent to release firmware supporting multiprotocol, which allows the device's Silicon Labs chip to connect to both Zigbee and Thread networks with one radio. <br /><br />
```
Should the product name be non-breaking?